app/assets/stylesheets/pages/code_csv_list_page.scss

```diff
@@ -22,6 +22,8 @@
 
 .name-date-wrapper {
   flex-flow: column;
+  width: 100%;
+  height: 100%;
 }
```

Comment:
I would change the `cursor: pointer` event to only be on `.name-date-wrapper` since you're using that to open the modal.